modules/migration-combining-must-gather.adoc

```diff
@@ -18,3 +18,10 @@ The only other variables that you can specify this way are the following:
 
 * `logs_since`, with a default value of `72h`
 * `request_timeout`, with a default value of `0s`
+
+If DataProtectionApplication (DPA) is configured with `s3Url` and has an `insecureSkipTLS` flag set to `true` it will not collect the necessary logs due to a missing CA certificate. In order to collect those logs run the `must-gather` command with the following option:
```

Comment:
```suggestion
If DataProtectionApplication (DPA) is configured with `s3Url` and `insecureSkipTLS: true`, the CR does not collect the necessary logs because of a missing CA certificate. To collect those logs, run the `must-gather` command with the following option:
```

Reply:
Thank you!!!